src/index.html

```diff
@@ -17,16 +17,30 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="form form_big"
+      action="search"
+      data-qa="big"
+    >
```

Comment:
Remember to keep your attributes correctly formatted. Each attribute should start on a new line with 2-space indentation. The closing bracket of the tag should be on the same level as the opening one.